doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -21,6 +21,155 @@ This document describes the changes required or recommended when migrating your
+      * Engineering C - v0.8.3 and later revisions
+
+      Check the version number on your DK's sticker to verify its compatibility with the |NCS|.
+
+Dependencies
+============
+
+The following required dependencies for the nRF54H20 SoC and DK have been updated.
+
+nRF54H20 BICR
+-------------
+
+.. toggle::
+
+  * The nRF54H20 BICR has been updated (from the one supporting |NCS| v2.7.0).
+
+    .. note::
+       BICR update is not required if migrating from |NCS| v2.7.99-cs1 or v2.7.99-cs2.
+
+    To update the BICR of your development kit while in Root of Trust, do the following:
+
+    1. Download the `BICR new binary file`_.
+    #. Connect the nRF54H20 DK to your computer using the **DEBUGGER** port on the DK.
+
+       .. note::
+          On MacOS, connecting the DK might repeatedly trigger a popup displaying the message ``Disk Not Ejected Properly``.
+          To disable this, run ``JLinkExe``, then run ``MSDDisable`` in the J-Link Commander interface.
+
+    #. List all the connected development kits to see their serial number (matching the one on the DK's sticker)::
+
+          nrfutil device list
+
+    #. Move the BICR HEX file to a folder of your choice, then program the BICR by running nRF Util from that folder using the following command::
+
+          nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware <path_to_bicr.hex> --core Application --serial-number <serial_number>
+
+nRF54H20 SoC binaries
+---------------------
+
+.. toggle::
+
+  * The *nRF54H20 SoC binaries* bundle has been updated to version 0.6.8.
+
+    .. caution::
+       If migrating from |NCS| v2.7.0, before proceeding with the SoC binaries update, you must first update the BICR as described in the previous chapter.
+
+    To update the SoC binaries bundle of your development kit while in Root of Trust, do the following:
+
+    1. Download the nRF54H20 SoC binaries v0.6.8:
+
+       * `nRF54H20 SoC Binaries v0.6.8 for EngC DKs`_, compatible with the nRF54H20 DK v0.8.3 and later revisions
+       * `nRF54H20 SoC Binaries v0.6.8 for EngB DKs`_, compatible with the nRF54H20 DKs ranging from v0.8.0 to v0.8.2.
+
+       .. note::
+          On MacOS, ensure that the ZIP file is not unpacked automatically upon download.
+
+    #. Purge the device as follows::
+
+          nrfutil device x-boot-mode-set --boot-mode safe --serial-number <serial_number>
+          nrfutil device erase --all --core Application --serial-number <serial_number>
+          nrfutil device erase --all --core Network --serial-number <serial_number>
+          nrfutil device x-boot-mode-set --boot-mode normal --serial-number <serial_number>
+
+    #. Run ``west update``.
+    #. Move the correct :file:`.zip` bundle to a folder of your choice, then run nRF Util to program the binaries using the following command::
+
+          nrfutil device x-suit-dfu --firmware nrf54h20_soc_binaries_v0.6.8_<revision>.zip --serial-number <serial_number>
```

Comment:
For `nrfutil device 2.7.2` and for `engb`, the command is OK. However, for `engc`, an update candidate address needs to be specified. The full command will be:
```
nrfutil device x-suit-dfu --firmware nrf54h20_soc_binaries_v0.6.8_<revision>.zip --serial-number <serial_number> --update-candidate-info-address 0x0e1ef340
```